qoffer-infrastructure/src/main/resources/offer-template/offer.html

```diff
@@ -62,7 +62,8 @@
         <div class="project-section">
             <div id="project-title">Project title</div>
             <div class="project-content-topic">Objective</div>
-            <div id="objective" class="project-content">Describe what the project is about here. Make sure not to write in too much detail as we want to prevent unauthorized people to get too much knowledge about the project.
+            <div id="project-objective" class="project-content">Describe what the project is about
```

Comment:
```suggestion
            <div id="project-objective" class="project-content">Describe what the project is about
```
:+1: greatly appreciate the hyphenated id name.